ssz/sedes/serializable.py

```diff
@@ -179,6 +179,17 @@ def __deepcopy__(self, *args):
     def root(self):
         return ssz.hash_tree_root(self)
 
+    def to_formatted_dict(self, int_as="int"):
```

Comment:
I've grown to treat magic strings as a code smell (error prone pattern).  I'd advocate for this to either become an `enum.Enum` or better, to pass around an `basic_sedes_formatter` that is responsible for doing the formatting of the basic types.

```python
class DefaultFormatter:
    @staticmethod
    def format_integer(value: int):
        return value
    @staticmethod
    def format_bool(value: bool):
        return value
```